lib/drivers/animasDriver.js

```diff
@@ -1647,6 +1749,79 @@ module.exports = function (config) {
     return postrecords;
   }
 
+  function buildCBGRecords(data, postrecords) {
+    data.bgRecords.forEach(function (event) {
+      var record = cfg.builder.makeDeviceEventCalibration();
+      record.with_value(event.value)
+            .with_deviceTime(event.deviceTime)
+            .with_units('mg/dL')      // hard-coded in mg/dL
+            .set('index', event.index);
+      cfg.tzoUtil.fillInUTCInfo(record, event.jsDate);
+      record = record.done();
+      postrecords.push(record);
+    });
+
+    var timeChanges = 0;
+
+    data.cbgRecords.forEach(function (event) {
```

Comment:
being paranoid but just wondering in these situations can the details in the event ever be null? e.g. `event.cbg_types` or `event.glucose`?